docs/Glossary.md

```diff
@@ -13,6 +13,16 @@ description: >
 
 The following terms are arranged in the order of their appearance in the actual user workflow.
 
+### Projects
+**A project is a set of related [Data Scopes](#data-scope), the Data Scopes may contains different [Data Entities](#data-entities) from arbitrary [Data Sources](#data-sources) for Metric Computation**
```

Comment:
1. Text change: "may contains" to "may contain"
2. Can you add an example to explain what is a concrete data scope/entities?  Something like "For example, a project collects the data from GitHub repo A&B and Jira board C. GitHub Repo A, GitHub repo B and Jira board C are three separate Data Scopes......"